crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -196,94 +193,16 @@ impl<S> Node<S>
 where
     S: Storage + Send + Sync + 'static,
 {
-    /// Constructs a new node to be run as the app
-    #[allow(clippy::too_many_arguments)]
-    pub fn new_app(
-        alias: &str,
-        network: Network,
-        data_dir: &Path,
-        node_storage: Arc<S>,
-        announcement_address: SocketAddr,
-        listen_address: SocketAddr,
-        esplora_server_url: String,
-        seed: Bip39Seed,
-        ephemeral_randomness: [u8; 32],
-        oracle: OracleInfo,
-    ) -> Result<Self> {
-        let ldk_config = app_config();
-        Node::new(
-            alias,
-            network,
-            data_dir,
-            node_storage,
-            announcement_address,
-            listen_address,
-            vec![util::build_net_address(
-                announcement_address.ip(),
-                announcement_address.port(),
-            )],
-            esplora_server_url,
-            seed,
-            ephemeral_randomness,
-            ldk_config,
-            LnDlcNodeSettings::default(),
-            oracle.into(),
-            disk::in_memory_scorer,
-        )
-    }
-
-    /// Constructs a new node to be run for the coordinator
-    ///
-    /// The main difference between this and `new_app` is that the user config is different to
-    /// be able to create just-in-time channels and 0-conf channels towards our peers.
-    #[allow(clippy::too_many_arguments)]
-    pub fn new_coordinator(
-        alias: &str,
-        network: Network,
-        data_dir: &Path,
-        node_storage: Arc<S>,
-        announcement_address: SocketAddr,
-        listen_address: SocketAddr,
-        announcement_addresses: Vec<NetAddress>,
-        esplora_server_url: String,
-        seed: Bip39Seed,
-        ephemeral_randomness: [u8; 32],
-        settings: LnDlcNodeSettings,
-        oracle: OracleInfo,
-    ) -> Result<Self> {
-        let mut ldk_config = coordinator_config();
-
-        // TODO: The config `force_announced_channel_preference` has been temporarily disabled
-        // for testing purposes, as otherwise the app is not able to open a channel to the
-        // coordinator. Remove this config, once not needed anymore.
-        ldk_config
-            .channel_handshake_limits
-            .force_announced_channel_preference = false;
```

Comment:
I guess the tests will reveal if we don't need this anymore.

Reply:
yeah, if the tests pass, it's probably not needed :)